books/free-programming-books-uk.md

```diff
@@ -19,19 +18,12 @@
 
 ### Haskell
 
-* [Вивчить собі Хаскела на велике щастя!](http://haskell.trygub.com) - Міран Ліповача
-
-
-### HTML / CSS
-
-#### Bootstrap
-
-* [Ознайомлення Bootstrap 3.3.2](http://twbs.docs.org.ua)
+* [Вивчити собі Хаскель на велике щастя!](http://haskell.trygub.com) - Міран Ліповача
 
 
 ### Java
 
-* [Програмування мовою Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
+* [Програмування на Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
```

Comment:
Why did you change the original name? 
![image](https://user-images.githubusercontent.com/12746043/195761209-db948916-4931-45f3-acd4-35416a9f4498.png)